docs/links/mautic_contribution_guide.py

```diff
@@ -2,6 +2,6 @@
 
 link_name = "Mautic Contribution guidelines" 
 link_text = "Mautic Contribution guidelines"
-link_url = "https://mau.tc/contribute" 
+link_url = "https://contribute.mautic.org/contributing-to-mautic" 
```

Comment:
The mau.tc link is still valid - it's a tracked URL we use with our yourls platform - just in case you weren't aware! The reason it's blocked is we have DDOS protection enabled, we need to exclude the domain from being checked by the link checker, ideally, as a longer term solution.

Reply:
Thanks for the info!

Later on, we might want to change this link into the internal link with `:docs:` as this is part of the Community Handbook page.